web/src/components/onboarding/ReturnToPreviousBusinessBar.tsx

```diff
@@ -20,7 +21,10 @@ export const ReturnToPreviousBusinessBar = (props: Props): ReactElement | null =
   const { updateQueue, userData } = useUserData();
   const { state } = useContext(AuthContext);
   const router = useRouter();
-  const businessName = getNavBarBusinessTitle(props.previousBusiness, state.isAuthenticated);
+  const businessName = getNavBarBusinessTitle(
```

Comment:
sand - this component would be a good candidate for a refactor where we inject domain logic via props and the would render as a dummy component. Thoughts?

Reply:
Sure? I think we could pass in all the data we're getting from useX calls here. I'll write a ticket for it for the future.